test_reporting/collect_azp_results.py

```diff
@@ -20,13 +20,27 @@ def get_tasks_results(buildid):
         dict: Dict of tasks' results
     """
     task_results = {
+        "start_time": "",
         "success_tasks": "",
         "failed_tasks": "",
         "cancelled_tasks": ""
     }
-    pipeline_url = "https://dev.azure.com/mssonic/internal/_apis/build/builds/" + str(buildid) + "/timeline?api-version=5.1"
-    print("Collect task results from here:{}".format(pipeline_url))
-    build_records = requests.get(pipeline_url, auth=AUTH).json()["records"]
+
+    pipeline_url = "https://dev.azure.com/mssonic/internal/_apis/build/builds/"+ str(buildid)
+    print("Collect pipeline startTime from here:{}".format(pipeline_url))
+    api_result = requests.get(pipeline_url, auth=AUTH)
+    starttime_str = api_result.json()["startTime"]
+
+    # Convert the time format from 2022-08-09T03:00:32.7088577Z
```

Comment:
this format is UTC, should we keep it?

Reply:
In kusto, datatime format should be like this 2022-08-09 03:00:32.7088577, if upload 2022-08-09T03:00:32.7088577Z, it doesn't work, it will be empty.